packages/hardhat-zksync-verify/src/zksync-block-explorer/service.ts

```diff
@@ -26,7 +26,13 @@ export async function checkVerificationStatusService(
     let verificationStatusResponse;
 
     try {
-        const data = await axios.get(`${verifyURL}/${requestId}`);
+        const url = new URL(verifyURL || '');
```

Comment:
Please move this logic to a separate util function please and then reuse.

Reply:
I have moved the logic to util. Can you help me to review again? Thanks~